src/resource/useWatchResourceValue.ts

```diff
@@ -18,6 +19,7 @@ export const useWatchResourceValue = <
     keepValueWhileLoading = true,
     useSuspense = true,
     refreshOnWindowFocus = false,
+    refreshOnVisibilityChange = false,
```

Comment:
Könnte mir vorstellen, dass das im Default von `refreshOnWindowFocus` abhängt. 

```suggestion
    refreshOnVisibilityChange = refreshOnWindowFocus,
```

Reply:
Jau, das kann man hier gut machen 👍